Calories.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep 23 00:45:03 2020
+
+@author: olumideakinola
+"""
+
+from clarifai.rest import ClarifaiApp, Image as ClImage
+
+import wolframalpha
+
+
+class Calories:
+ 
+    
+    def __init__(self,image):
+        self.image = image
+    
+    def getFoodName(self):
+        self.lst=[]
+        app = ClarifaiApp(api_key='64f97bc1c2e34da9aeaf6103fea866d4')
```

Comment:
`api_key` should also be passed an environment variable. See comment in flask2.py

Reply:
done